src/commonMain/kotlin/fr/acinq/lightning/wire/ChannelTlv.kt

```diff
@@ -15,14 +15,11 @@ import fr.acinq.lightning.utils.msat
 import fr.acinq.lightning.utils.sat
 import fr.acinq.lightning.utils.toByteVector
 import fr.acinq.lightning.utils.toByteVector32
-import kotlinx.serialization.Contextual
-import kotlinx.serialization.Serializable
 
-@Serializable
 sealed class ChannelTlv : Tlv {
     /** Commitment to where the funds will go in case of a mutual close, which remote node will enforce in case we're compromised. */
-    @Serializable
-    data class UpfrontShutdownScriptTlv(@Contextual val scriptPubkey: ByteVector) : ChannelTlv() {
+
```

Comment:
nit: remove empty line between documentation and class?

Reply:
Done in c5365f4.